From fba54a3e3bb85857dc3fdaba2e6cbf792d4dc1fa Mon Sep 17 00:00:00 2001
From: YmrDtnJu <YmrDtnJu@users.noreply.github.com>
Date: Fri, 22 Jun 2018 23:17:07 +0000
Subject: [PATCH 01/25] core: Actually use the resolved path for
 TemporaryFileSystem= (#9385)

The code already resolves specifiers using unit_full_printf() but then uses the
unresolved version again for temporary_filesystem_add().

(cherry picked from commit a26fec240862c7bc466f468490bae2395f263708)
---
 src/core/load-fragment.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/load-fragment.c b/src/core/load-fragment.c
index d9a5094aa0..fa3605294b 100644
--- a/src/core/load-fragment.c
+++ b/src/core/load-fragment.c
@@ -3904,7 +3904,7 @@ int config_parse_temporary_filesystems(
                 if (r < 0)
                         continue;
 
-                r = temporary_filesystem_add(&c->temporary_filesystems, &c->n_temporary_filesystems, path, w);
+                r = temporary_filesystem_add(&c->temporary_filesystems, &c->n_temporary_filesystems, resolved, w);
                 if (r == -ENOMEM)
                         return log_oom();
                 if (r < 0) {
-- 
2.19.1

